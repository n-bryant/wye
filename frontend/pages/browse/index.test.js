import { BrowsePage } from "./index";

describe("BrowsePage", () => {
  const props = {
    classes: {
      root: "root",
      container: "container",
      main: "main",
      heading: "heading",
      subHeading: "subHeading"
    }
  };

  it("should render successfully", () => {
    expect(toJson(shallow(<BrowsePage {...props} />))).toMatchSnapshot();
  });
});
