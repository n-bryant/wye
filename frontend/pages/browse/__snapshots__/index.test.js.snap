// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrowsePage should render successfully 1`] = `
<BackgroundQuery>
  <div
    className="wye-BrowsePage root"
  >
    <WithStyles(ForwardRef(Container))
      className="wye-BrowsePage__container container"
      disableGutters={true}
      maxWidth="lg"
    >
      <WithStyles(ForwardRef(Container))
        className="wye-BrowsePage__main main"
        maxWidth={false}
      >
        <WithStyles(ForwardRef(Typography))
          className="wye-BrowsePage__heading heading"
          variant="h1"
        >
          Featured Games
        </WithStyles(ForwardRef(Typography))>
        <WithStyles(ForwardRef(Typography))
          className="wye-BrowsePage__subHeading subHeading"
          variant="h2"
        >
          Browse the most popular and discounted multiplayer games
        </WithStyles(ForwardRef(Typography))>
        <MostPopularFeaturedQuery />
      </WithStyles(ForwardRef(Container))>
    </WithStyles(ForwardRef(Container))>
  </div>
</BackgroundQuery>
`;
